makefile

```diff
@@ -17,14 +17,24 @@ REQUIRE = requirements.txt
 ext_py := $(shell which python3 || which python)
 
 # Override by putting in commandline python=XXX when needed.
+<<<<<<< HEAD
+python = $(shell echo ${ext_py} | rev | cut -d '/' -f 1 | rev)
+TESTDIR = $(shell realpath tests)
+ENV = $(shell realpath venv)
```

Comment:
this doesn't work locally on my machine

Reply:
do you use windows? (I pushed fix for this)

Reply:
nope, I'm on mac

Reply:
weird/ Did I understand you right that there is no `realpath` cmd in mac env?

Reply:
if so, I think this cmd will install it - `brew install coreutils`